src/components/lib/cos_kernel_api.c

```diff
@@ -509,6 +509,21 @@ cos_asnd_alloc(struct cos_compinfo *ci, arcvcap_t arcvcap, captblcap_t ctcap)
 	return cap;
 }
 
+
+hwcap_t
```

Comment:
maybe a TODO here concerning the fact that this breaks access control, and in the future we need to make it so that you can only crate a hwcap from an existing one.